.github/workflows/main.yml

```
@@ -26,6 +26,19 @@ jobs:
         python-version: '3.7'
         architecture: 'x64'
     - name: Install dependencies
-      run: source ./ci_install.sh
-    - name: Run tests
-      run: source ./ci_script.sh
+      run: |
+        python -m pip install jupyterlab~=3.0.0rc2
```

Since rc7 contains this fix: https://github.com/jupyterlab/jupyterlab/pull/9272

We can update CI to see if the build passes:

```suggestion
        python -m pip install jupyterlab~=3.0.0rc7
```